### DISCUSSION TITLE
Unspecified Deployment Error with Next.js on Vercel!

### DISCUSSION THREAD

**Discussion by arman-mokhtari** [vercel/next.js#62862] [Help]
### Summary

When attempting to deploy, I encountered the following error message: "Your Team encountered an unknown problem. Please reach out to our support team for details." Unfortunately, the error message does not provide any specific details regarding the nature of the problem, making it difficult for me to troubleshoot on my own.

I have ensured that all dependencies are properly installed and that my Next.js project is configured correctly. Despite these efforts, I continue to encounter the same error message.

**Comment by icyJoseph** [comment]
Hi, 

Have you contacted their customer support though? Typically they answer rather fast, and can give far more insight into this kind of "account" specific issues.

**Reply by arman-mokhtari** [reply]
Resolved by deleting Vercel account and registering again.
